code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -11,7 +11,7 @@
 	mutant_organs = list(/obj/item/organ/tail/cat)
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | RACE_SWAP | ERT_SPAWN | SLIME_EXTRACT
 	species_language_holder = /datum/language_holder/felinid
-	disliked_food = CLOTH
+	disliked_food = GROSS | RAW | CLOTH
```

Comment:
this will make felinids dislike eating mice again, which is kind of a shame. even if we add a "feline" food type later and add it to the food types of dead mice, felinids will still dislike them, as the disliked_food list takes priority over the liked_food list.

Reply:
This wasn't the intention of the pr that changed it, so I do not care.